最新のLHAを構築中のエラー

最新の `lha`をビルドしようとすると、このエラーが表示されます。
```
/usr/local/bin/ld: header.o: in function `ConvertEncodingByIconv':
/usr/local/tmp/crew/6f6cbc1.tar.gz.dir/lha-6f6cbc189d23b7c3a8636819f5796654f188764e/src/header.c:2024: undefined reference to `libiconv_open'
/usr/local/bin/ld: /usr/local/tmp/crew/6f6cbc1.tar.gz.dir/lha-6f6cbc189d23b7c3a8636819f5796654f188764e/src/header.c:2030: undefined reference to `libiconv'
/usr/local/bin/ld: /usr/local/tmp/crew/6f6cbc1.tar.gz.dir/lha-6f6cbc189d23b7c3a8636819f5796654f188764e/src/header.c:2038: undefined reference to `libiconv_close'
/usr/local/bin/ld: /usr/local/tmp/crew/6f6cbc1.tar.gz.dir/lha-6f6cbc189d23b7c3a8636819f5796654f188764e/src/header.c:2032: undefined reference to `libiconv_close'
collect2: error: ld returned 1 exit status
```
これが機能するためには `libiconv`とリンクする必要があるようです。

これが紛らわしい場合は申し訳ありませんが、私は翻訳者を使用しています。

--- a/configure.ac
+++ b/configure.ac
@@ -58,6 +58,7 @@ fi
 
 # Checks for libraries.
 AC_SEARCH_LIBS(opendir, [mingwex])
+AC_SEARCH_LIBS(libiconv, [iconv])
 AC_CHECK_LIB(applefile, af_open)
 
 # Checks for header files.
@@ -65,6 +66,7 @@ AC_HEADER_DIRENT
 AC_HEADER_STDC
 AC_CHECK_HEADERS(fcntl.h limits.h sys/file.h sys/param.h sys/time.h)
 AC_CHECK_HEADERS(pwd.h grp.h utime.h inttypes.h stdint.h fnmatch.h)
+AC_CHECK_HEADERS(iconv.h)
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST

